assets/Timo_Daum-Agile_Methods_(English).md

```diff
@@ -0,0 +1,86 @@
+
+Furthermore, the team members are burdened with additional "affective work" by the agile working methods, they have to cope with the technical challenges and the sustainable pace of the team. "Workers are expected to self-manage the impact of constant change through emotional management and affective control. Managing change thus becomes an all-of-life responsibility, where wellbeing is a worker’s remit." writes researcher Phoebe Moore. [13]
+
+#### The whole life as a project
+
+Management methods penetrate our private lives and force us into roles that originate in project management. The waterfall model as a project management ideal has worn out, and biographies are also becoming increasingly non-linear; agility is demanded in all situations in life, not excluding senior citizens. They as well must always be active, the well-deserved retirement from the past days of fordist biographies with a clear sequence of events has become obsolete. The project has long since left the domain of work - the whole life becomes a project and managed in an agile way. Self improvement literature provides the soundtrack for this, "Emotional Agility", as the book title by life coach Susan David proclaims agile methods for self-management: "get unstuck, embrace change, and thrive in work and life" [14]
+
+In the course of the "new spirit of capitalism", self-optimization, lifelong learning, entrepreneurial validation of one's own work and biography become a constant companion for every individual. Charles Hardy, the philosopher and author of The Age of Unreason, had similar thoughts back in the 1980s when he proposed "replacing the traditional concept of work with the concept of a portfolio of areas of activity in which everyone is active for their own account." [15]  The project replaces the factory, the self-entrepreneur replaces the employee and makes the further development and improvement of one's own employability a matter for the boss: "Understanding oneself as a company and looking at it through the eyes of potential customers" - this is how sociologist Ulrich Bröckling described this new self-image as early as 2004.
+
+In all situations of life we are now called upon to measure ourselves, to take action, to take risks, to become our own CEO. As individuals, we are called upon to look at ourselves as entrepreneurs. Our own self becomes human capital and must consequently be optimised in terms of business management: Staying on a diet, going to the gym - no aspect of life is left out when optimizing. Quantified Self is the realization of Taylor's program of a scientific management of the resource of living labor, this time shifted into the individual.
```

Comment:
> As individuals, we are called upon to look at ourselves as entrepreneurs.

I would replace "called upon" with "asked". Called upon is way too strong given the context.

Reply:
> Staying
The S should be lowercase